modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -97,8 +81,65 @@ import canvas from '@workday/canvas-kit-react/tokens';
 ```
 
-This change is covered by the codemod, but it doesn't currently run on files that don't use the
-`.js`, `.jsx`, `.ts`, and `.tsx` extensions. Those files will have to be updated manually.
+🤖 The codemod will update import statements to use the new slash imports syntax.
+
+Recall that the codemod only works on `.js`, `.jsx`, `.ts`, and `.tsx` extensions. Other file types
+will need to be updated manually.
+
+### Canvas Kit Preview & Component Promotions
+
+Due to the broad range of stability in [Canvas Kit Labs](../../labs-react)
+(`@workday/canvas-kit-labs-react`), we're introducing a new module called
+[Canvas Kit Preview](../../preview-react) (`@workday/canvas-kit-preview-react`) to provide consumers
+with more clarity and confidence when uptaking experimental and upcoming components. The components
+in Preview have had a full design and accessibility review and are approved for use in product.
+Their functionality and design are set, but their APIs and/or underlying architecture are still
+subject to change.
+
+Preview serves as a staging ground for components that are ready to use, but may not be up to the
+high code standards upheld in the Main `@workday/canvas-kit-react` module. Think of Labs as a space
+for alpha components and Preview as a space for beta components.
+
+We've promoted several components from Labs to Preview in v5:
+
+#### Promoted from Labs to Preview
+
+- Breadcrumbs
+- Color Picker
+- Core `type` > Tokens `type`
+- Menu
+- Select
+- Side Panel
+
+Generally, a component will begin in Labs before it's promoted to Preview and eventually to Main
+(although there is no guarantee a component will advance out of Labs). Given that Preview was just
+introduced here in v5, however, we believe that a few components in Labs are ready for Main and thus
+we've promoted them straight from Labs to Main:
+
+#### Promoted from Labs to Main
+
+- Pagination
+- Tabs
+
+### Rename Core and Labs Core Modules to Tokens
+
+The distinction between our core and common packages is often unclear and creates confusion around
+what should be imported from where. To help alleviate this and better align with our design
+taxonomy, we're renaming our `core` packages to `tokens`. We're also removing the `type` default
+export in the labs core package in favor of named imports. These changes are listed below, all of
+which are handled by the v5 codemod.
+
+#### Automatic Updates
+
+- 🤖 Core import statements
+  - e.g. `import { space } from '@workday/canvas-kit-react/core';` becomes
+    `import { space } from '@workday/canvas-kit-react/tokens';`
+- 🤖 Labs core import statements
+  - e.g. `import { type } from '@workday/canvas-kit-labs-react/core';` becomes
+    `import { type } from '@workday/canvas-kit-labs-react/tokens';`
+- 🤖 Labs core default type import
+  - e.g. `import type from '@workday/canvas-kit-labs-react/core';` becomes
+    `import { type } from '@workday/canvas-kit-labs-react/tokens';`
```

Comment:
Some of these changes regarding the type hierarchy and tokens have already been obsoleted by later changes in v5 (for example `labs/tokens` was removed in #1081). Not going to change anything here now, though, since the migration guide will be updated in #1078 anyway.